src/unix/process.c

```diff
@@ -180,6 +180,8 @@ int uv__make_pipe(int fds[2], int flags) {
   return 0;
 }
 
+/* Verify that HANDLE is same as int*/
+STATIC_ASSERT(sizeof(((uv_stdio_container_t*)0)->data.handle) == sizeof(((uv_stdio_container_t*)0)->data.fd));
```

Comment:
Can you keep lines <= 80 columns?

Also, in the comment above, can you end with `...is same as int. */`, i.e., punctuation and a blank?